src/main/java/net/dv8tion/jda/api/hooks/ListenerAdapter.java

```diff
@@ -573,6 +580,8 @@ else if (event instanceof GenericVoiceChannelUpdateEvent)
             onGenericVoiceChannelUpdate((GenericVoiceChannelUpdateEvent) event);
         else if (event instanceof GenericGuildUpdateEvent)
             onGenericGuildUpdate((GenericGuildUpdateEvent) event);
+        else if (event instanceof GenericGuildMemberUpdateEvent)
+            onGenericGuildMemberUpdateEvent(((GenericGuildMemberUpdateEvent) event));
         else if (event instanceof GenericGuildMemberEvent)
             onGenericGuildMember((GenericGuildMemberEvent) event);
```

Comment:
The check for `onGenericGuildMember` needs to be moved to the if-block for generic superclasses.

It starts under `//Generic events that have generic subclasses (the subclasses as above).`

Reply:
Thanks for the heads up! Moved it.